tool/tctl/common/fido2.go

```diff
@@ -0,0 +1,39 @@
+// Teleport
+// Copyright (C) 2024 Gravitational, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package common
+
+import (
+	"context"
+
+	"github.com/alecthomas/kingpin/v2"
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/service/servicecfg"
+	"github.com/gravitational/teleport/tool/common/fido2"
+)
+
+// fido2Command adapts fido2.Command for tctl.
+type fido2Command struct {
+	impl *fido2.Command
+}
+
+func (c *fido2Command) Initialize(app *kingpin.Application, _ *servicecfg.Config) {
```

Comment:
I've added the wrappers here so we avoid importing both `servicecfg` and `lib/auth` in the "common" packages. The former is not imported by tsh, so that should keep binary sizes as-is.

Same for the other wrappers.